mapedit/npcedit.cc

```diff
@@ -437,8 +437,8 @@ int ExultStudio::init_npc_window(
 	// Store address with window.
 	g_object_set_data(G_OBJECT(npcwin), "user_data", addr);
 	// Store name, ident, num.
-	const utf8Str utf8name(name.c_str());
-	set_entry("npc_name_entry", utf8name);
+	std::string utf8name(convertToUTF8(name.c_str()));
```

Comment:
This applies in general, but picking this specific case: why not `const string`?